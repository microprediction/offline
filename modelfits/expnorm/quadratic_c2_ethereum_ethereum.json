{"g1": 0.3820478350836981, "g2": 0.06586895630906553, "logK": 0.9010803956816902, "loc": 0.07930897600976108, "logScale": -0.34165735331795116}