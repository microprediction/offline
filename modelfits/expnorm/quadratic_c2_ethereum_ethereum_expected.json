{"g1": 0.9572183687592055, "g2": 6.952107348414458, "logK": 0.9844312541735065, "loc": -0.08715690489024235, "logScale": -1.5267688872456027}